modules/ROOT/pages/migrate-from-an-earlier-version-of-bonita-bpm.adoc

```diff
@@ -561,3 +561,66 @@ UPDATE QRTZ_TRIGGERS SET TRIGGER_STATE = 'WAITING' WHERE TRIGGER_STATE = 'PAUSED
 ----
 
 After this operation, the table QRTZ_PAUSED_TRIGGER_GRPS should be empty, and all the triggers in the QRTZ_TRIGGERS table should be in state _waiting_, and not _paused_.
+
+[#fk_troubleshoot]
+=== Some foreign keys are duplicated
+
+_Symptom_: After migrating to 7.11.0+, prior to 7.11.6, after re-installing/updating the BDM, some foreign key constants are effectively duplicated.
+
+_Cause_: A bug in an underlying product dependency (hibernate: )The upgrade of an internal library, from Hibernate 4 to 5, introduces a bug ()
+
+_Solution_: Doing nothing is an option, as there is no change in product behaviour. If you wish to clean your BDM database, follow the procedure:
+
+
+1. Stop your bonita server
+
+1. Open the db in an edition tool (or execute in command line the relevant commands):
+First, select all the foreign key on a table:
+
+Mysql:
```

Comment:
⚠️ Code samples do not show as Code samples:
https://bonitasoft-bonita-doc-build_preview-pr-1512.surge.sh/bonita/7.11/migrate-from-an-earlier-version-of-bonita-bpm#fk_troubleshoot

Reply:
See SQL code sample here, for instance
https://documentation.bonitasoft.com/bonita/2021.1/purge-tool-deletion-volume-testing#_ms_sql_server